packages/conf-gfortran/conf-gfortran.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Virtual package relying on a gfortran system installation"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "The GCC team"
+license: "GPL 3"
+homepage: "https://gcc.gnu.org/fortran/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+flags: conf
+build: ["gfortran" "--version"]
```

Comment:
Are you sure all external dependencies install `gfortran`?  Odepack `discover` script also looks for `g95` and `g77`.

Reply:
I looked at the packages for freebsd and openbsd and they seem wrong (it installs something unrelated). I'll fix this but the rest seems to install `gfortran` "at least"